### DISCUSSION TITLE
artifactory_api_key deprecation for Cloud artifactory

### DISCUSSION THREAD

**Discussion by evilensky** [jfrog/terraform-provider-artifactory#614] [General] [answered]
Hello,

We got in touch with our jfrog account team to clarify the timeline for artifactory_api_key deprecation for Cloud customers. Their response is roughly, that this is not scheduled for Cloud customers at all and they will communicate as soon as they hear more.

The question in this discussion is, should artifactory Cloud customers expect this provider to continue to work with `artifactory_api_key` for as long as its supported by cloud artifactory?

Thank you!

**Comment by alexhung** [comment, marked as answer]
@evilensky That's correct. We are not removing API key support until Artifactory removes it. However, having said that, I still strongly recommend you to migrate to access token now (vs later).

cc @danielmkn
